<launch>
  
  <node pkg = "master_package" type = "master_node.py" name = "master_node" output = "screen"/>
  <node pkg = "positioning_package" type = "positioning_node.py" name = "positioning_node" output = "screen"/>
  <node pkg = "mission_package" type = "mission_node.py" name = "mission_node" output = "screen"/>
  <node pkg = "cv_package" type = "detect.py" name = "cv_node" output = "screen"/>
  <node pkg = "movement_package" type = "movement_node.py" name = "movement_node" output = "screen"/>
  
</launch>
